### PR TITLE
ASPECT publications are deal.II publications as well.

### PR DESCRIPTION
**Pull request by bangerth**
/rebuild

### CODE REVIEW

**Comment by marcfehling** [comment]
Hmm, do you want to keep them in separate documents/separate folders?

This way it would be easier to maintain the aspect part of deal.II publications -- you could just copy over whatever you gathered in the aspect project.

File: publications-2014.bib [hunk 1]
```diff
@@ -1388,4 +1388,21 @@ @article{Albert2014
...
+@phdthesis{Quinquis2014,
+author = {Quinquis, M.},
+school = {Charles University},
+title = {{A numerical study of subduction zone dynamics using linear viscous to thermo-mechanical model setups including (de)hydration processes}},
+year = {2014}
+}
+
+@mastersthesis{Fraters2014,
+author = {Fraters, Menno},
+file = {:home/heister/.local/share/data/Mendeley Ltd./Mendeley Desktop/Downloaded/Fraters - 2014 - Thermo-mechanically coupled subductionmodelling with ASPECT(2).pdf:pdf},
```
Comment (drwells) [inline publications-2014.bib:1400]:
```suggestion
```

**Comment by bangerth** [comment]
We already maintain ASPECT publications elsewhere. This just copies over what wasn't already listed among the deal.II publications.

**Comment by bangerth** [comment]
@drwells So fixed!

**Review by masterleinad — APPROVED** [review]

File: publications-2019.bib [hunk 2]
```diff
@@ -1961,6 +2064,17 @@ @article{Khler2019
...
+@article{Steinberger2019,
+author = {Steinberger, Bernhard and Bredow, Eva and Lebedev, Sergei and Schaeffer, Andrew and Torsvik, Trond H},
+journal = {Nature Geoscience},
+number = {1},
+pages = {61},
+publisher = {Nature Publishing Group},
+title = {Widespread volcanism in the {G}reenland-{N}orth {A}tlantic region explained by the {I}celand plume},
+volume = {12},
+year = {2019}
+}
```
Comment (marcfehling) [inline publications-2019.bib:2076]: https://doi.org/10.1038/s41561-018-0251-0

**Review by marcfehling — CHANGES_REQUESTED** [review]

**Comment by bangerth** [comment]
So fixed.

File: publications-2019.bib [hunk 3]
```diff
@@ -1,5 +1,31 @@
 % Encoding: US-ASCII
 
+@article{Heron2018,
```
Comment (marcfehling) [inline publications-2019.bib:3]:
```suggestion
@article{Heron2019,
```

**Review by marcfehling — APPROVED** [review]

File: publications-2019.bib (hunk 3 above)
Comment (bangerth) [reply publications-2019.bib:3]: Let's not worry about this and save the time and energy to change it -- we ignore these tags anyway when building the website.

**Pull request merged**